helm/tweek/templates/editor.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: editor
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: editor
+    spec:
+      containers:
+      - name: editor
+        image: soluto/tweek-editor:1.0.0-rc5
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 3000
+        env:
+        - name: REACT_APP_GATEWAY_URL
```

Comment:
There's no need for react_app_gateway_url if the editor is served through the gateway